google/cloud/policytroubleshooter/v1/.repo-metadata.json

```diff
@@ -3,11 +3,11 @@
     "api_shortname": "policytroubleshooter",
     "client_documentation": "https://cloud.google.com/cpp/docs/reference/policytroubleshooter/latest",
     "distribution_name": "google-cloud-cpp",
-    "issue_tracker": "https://issuetracker.google.com/savedsearches/6137148",
+    "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:690790%20status=open",
     "language": "cpp",
     "library_type": "GAPIC_AUTO",
     "name_pretty": "Policy Troubleshooter API",
-    "product_documentation": "https://cloud.google.com/iam/docs/troubleshooting-access",
+    "product_documentation": "https://cloud.google.com/policy-intelligence/docs/troubleshoot-access",
```

Comment:
This suggests we should fix the README.md and main.dox pages too, but that could be done in a followup PR.

Reply:
Or maybe we should automate that step ...

Reply:
Taking the first suggestion for now, but fixed them here.